src/odb/src/db/dbModInst.cpp

```diff
@@ -157,10 +157,18 @@ dbIStream& operator>>(dbIStream& stream, _dbModInst& obj)
   stream >> obj._group;
   // User Code Begin >>
   dbBlock* block = (dbBlock*) (obj.getOwner());
-  _dbDatabase* db = (_dbDatabase*) (block->getDataBase());
-  if (db->isSchema(db_schema_update_hierarchy)) {
+  _dbDatabase* _db = (_dbDatabase*) (block->getDataBase());
```

Comment:
Local variables should not have a leading _ as that denotes class members.

Reply:
done